.cirrus.yml

```diff
@@ -151,11 +151,18 @@ build_and_release_dotnet_task:
     PROJECT_DIR: ${CIRRUS_WORKING_DIR}\\sonar-text-dotnet
     SOLUTION_DIR: ${PROJECT_DIR}\\SonarLint.Secrets.DotNet.sln
     PIPELINE_SCRIPTS_DIR: ${PROJECT_DIR}\\pipeline_scripts
+    PROJECT_NAME: "sonar-secrets-dotnet"
```

Comment:
In multiple scripts, we switch to PowerShell. Can we set it to be the default by adding `CIRRUS_SHELL: powershell` env variable.

Reply:
We need bash to run the `source cirrus-env` commands.
They are bash scripts and it might be problematic to work in PowerShell for this case.